cmd/manager/aggregator.go

```diff
@@ -336,6 +336,31 @@ func createOrUpdateOneResult(crClient aggregatorCrClient, owner metav1.Object, l
 	return nil
 }
 
+func setScanTimestamp(crClient aggregatorCrClient, owner metav1.Object, annotations map[string]string) map[string]string {
+	// Get the scan timestamp
+	scan := &compv1alpha1.ComplianceScan{}
+	err := backoff.Retry(func() error {
+		err := crClient.getClient().Get(context.TODO(), types.NamespacedName{
+			Namespace: owner.GetNamespace(),
+			Name:      owner.GetName(),
+		}, scan)
+		if err != nil {
+			return err
+		}
+		return nil
+	}, backoff.WithMaxRetries(backoff.NewExponentialBackOff(), maxRetriesForTimestamp))
+	if err != nil {
+		cmdLog.Error(err, "Failed to get scan", "ComplianceScan.Name", owner.GetName())
+		return annotations
+	}
+	if annotations == nil {
+		annotations = make(map[string]string)
+	}
+
+	annotations[compv1alpha1.LastScannedTimestampAnnotation] = scan.Status.StartTimestamp.Format(time.RFC3339)
```

Comment:
Do we want to allow the last scanned time to be *before* the creation time?

```
$ oc get ccr ocp4-cis-node-worker-file-groupowner-multus-conf  -o json | jq .metadata.annotations
{
  "compliance.openshift.io/last-scanned-timestamp": "2024-08-15T22:17:08Z",
  "compliance.openshift.io/rule": "file-groupowner-multus-conf"
}
$ oc get ccr ocp4-cis-node-worker-file-groupowner-multus-conf  -o json | jq .metadata.creationTimestamp
"2024-08-15T22:17:40Z"
```

Reply:
that's a good question, but we need to have a way to associate scan with CCR, and we will not have endTimeStamp at time of updating/creating CCR

Reply:
I suppose if we waited until we had an endtime stamp from the ComplianceScan object we'd need to reiterate the entire list of CCRs again to set the `last-scanned-timestamp`.

Reply:
but how would are we able to track down if a scan is linked to a scan that way, don't we need some way to be able to have a unique idenfier for CCR and Scan object, in this case a start timestamp

Reply:
But I do see the point that it might not make sense to user that the last timestamp is before the creation time

Reply:
To summarize - the CCR scan timestamps will match the ComplianceScan timestamp. The creation time of the CCR will potentially be later than the scan timestamp. This should only be the case on the first scan, since rescans will cause the CCR to have an updated scan time.

We can highlight that detail in CRD documentation.